solvers/tv_mm.py

```diff
@@ -0,0 +1,37 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    from benchmark_utils.tv_numba import tv_mm
+
+
+class Solver(BaseSolver):
+    """Solve the TV minimization problem using a majorization-minimization algorithm.
+
+    I. Selesnick, “Total Variation Denoising (an MM algorithm)”.
+    https://eeweb.engineering.nyu.edu/iselesni/lecture_notes/TVDmm/
+    """
+
+    name = "tv_mm"
+    stopping_strategy = "iteration"
+    parameters = {"tol": [1e-3, 1e-4, 1e-5, 1e-6]}
```

Comment:
I don't think this makes sense to have tol as a parameter. You should set tol super tiny and just rely on n_iter param of run method to get the curve